src/main/java/racing/Cars.java

```diff
@@ -15,4 +17,18 @@ public Cars(List<Car> cars) {
     public Iterator<Car> iterator() {
         return cars.iterator();
     }
+
+    public List<Car> getWinnerList() {
+        return  cars.stream()
+                .filter(car -> car.getPosition() == maxPosition())
```

Comment:
@maxPosition은 매번 호출하지 않고 한번만 호출해 두면 어떨까요?
`int maxPosition = maxPosition();`

>최대한 getter 없이 구현해보려고 했는데 한계에 부딪혀 값 객체에만 getter를 사용하는 전략으로 수정하였습니다.

getter를 활용하지 않고 메시지를 보내면 어떨까요?
```suggestion
                .filter(car -> car.samePosition(maxPosition))
```

우려하신 것처럼 getter가 존재하는 것만으로도 의도와 다른 방향으로 사용될 수 있는데요 
이 부분은 개발자가 최대한 의식적으로 노력을 해야하는 부분이라고 생각합니다